mass_mailing_statistics_af/models/mass_mailing.py

```diff
@@ -47,9 +49,15 @@ def _compute_statistics(self):
                 row['opened_ratio'] = 100.0 * row['opened'] / total
                 row['replied_ratio'] = 100.0 * row['replied'] / total
                 row['bounced_ratio'] = 100.0 * row['bounced'] / total
+            if row['opened'] != 0:
+                row['ctor'] = 100.0 * row['clicks'] / row['opened']
             self.browse(row.pop('mailing_id')).update(row)
 
     def _compute_clicks_ratio_percentage(self):
         for rec in self:
             rec.clicks_ratio_percentage = "{0} {1}".format(rec.clicks_ratio, "%")
 
+    def _compute_ctor_percentage(self):
+        for rec in self:
+            rec.ctor_percentage = "{0} {1}".format(rec.ctor, "%")
```

Comment:
I would argue that f"{rec.ctor} %" is some what more clear on what you want to accive here
or
rec.ctor_percentage = "{0} %".format(rec.ctor)

Please fix on this row and on the row 54/58